calcom/cal.com
# feat: download insights raw data as csv

## What does this PR do?

- Adds button to insights that triggers new api endpoint that return file download of same data used for all the insights graphs.

Fixes #10747 

![image](https://github.com/calcom/cal.com/assets/4461358/14730655-701d-47c4-af1e-46e814873edc)


## Requirement/Documentation

## Type of change




## How should this be tested?

- Are there environment variables that should be set? No
- What are the minimal test data to have? using yarn seed-insights from packages/playwright would be useful to have an entire insights setup with bookings/users/event-types ready.
- What is expected (happy path) to have (input and output)? Download data that was used for the same count for the graphs
- Any other important info that could help to test that PR

## Mandatory Tasks
